client/src/ui/css/Gauge.module.css

```diff
@@ -0,0 +1,69 @@
+@import "../../index.css";
```

Comment:
Just curious - why do we need to import this here?

Reply:
I believe this is necessary to use CSS variables, i.e. `var(--regular-weight);`